packages/rest/src/providers/reject.provider.ts

```diff
@@ -24,7 +31,8 @@ export class RejectProvider implements Provider<Reject> {
   action({request, response}: HandlerContext, error: Error) {
     const err = <HttpError>error;
     const statusCode = err.statusCode || err.status || 500;
-    writeErrorToResponse(response, err);
+    const writeErrorToResponse = errorHandler(this.errorHandlerOptions);
```

Comment:
Creating a new handler function every time an error is handled is suboptimal performance wise. V8 has to allocate a new closure and create a new dynamic function, which puts more pressure on the garbage collector. Additionally, V8 may not be able to optimize the handler function when it's always created anew (it definitely used to be a problem back in CrankShaft days, before Node.js 8.3 upgraded to a TurboFan-enabled version of V8).

That's why I proposed to modify strong-error-handler to export a function accepting `(err, req, res, options)`. 

Another reason is that technically, the error handler middleware accepts also a `next` callback and I am not sure if it's a good idea to rely on the fact that the middleware never calls `next`.

Sorry for not providing a detailed explanation from the beginning.

Reply:
> Please note that we should always set `log: false`, otherwise we may end up with two logs for each failed request (see `this.logError` below).

On the second thought, if we refactor strong-error-handler to export a new function that can be used directly from LB4, then I think a better alternative is to leave logging out of this newly exported function. That way we don't have to worry about `log` flag because it will be always ignored.

Thoughts?